### PR TITLE
add slic3r directory with documentation and configs

### PR DESCRIPTION
**Pull request by bmatsuo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,36 +18,39 @@ Documentation
...
 their environment's PATH.
 
 ./build.sh
 
-Slicing API
------------
+On your server machine, install [slic3r](http://slic3r.org/download), the
+backend slicing software, onto the machine you want to act as your server.
+Then create a directory containing the Slic3r configuration files you would
+like clients to have make available.  See the Slic3r [doc](slic3r/README.md)
```
Comment (vjcharles) [inline README.md]: make->made

File: README.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 Matching Snuggies
 =================
 
-Matching Snuggies is a slicing software that exposes a backend slicing program
-(currently slic3r) through an HTTP API.  A command line slicing tool is
-provided for ease of use and to support eventual integration with host software
-like Repetier-Host and OctoPrint.
+Matching Snuggies is 'slicing' software that converts 3D models into G-code
```
Comment (vjcharles) [inline README.md:4]: mention `(.stl, .amf)`?

**Comment by vjcharles** [comment]
looks good overall. clear, concise intro. install is clear

**Comment by bmatsuo** [comment]
damn this was never merged

**Pull request closed without merging**